airflow/config_templates/config.yml

```diff
@@ -249,6 +249,14 @@
       type: string
       example: ~
       default: "0"
+    - name: default_task_retry_delay
+      description: |
+          The number of seconds each task is going to wait by default between retries. Can be overridden at
+          dag or task level.
```

Comment:
This should mention the value is in seconds.

Reply:
@uranusjr the description of the setting does mention seconds. Are you referring to the setting name itself, like `default_task_retry_delay_sec`?

Reply:
Recovery

Reply:
Oh, sorry I missed it.